php56/root/etc/confd/templates/xdebug.ini.tmpl

```diff
@@ -8,12 +8,15 @@ xdebug.coverage_enable=0
 xdebug.default_enable=1
 
 xdebug.remote_enable=1
+xdebug.remote_autostart=1
 xdebug.remote_connect_back=1
-xdebug.remote_host=localhost
+; If the following remote_host value does not work use the environmental
+; variable XDEBUG_CONFIG to override it. Documentation for format at
+; https://xdebug.org/docs/remote
+xdebug.remote_host=192.168.99.1
 xdebug.remote_port=9000
```

Comment:
Given "xdebug.remote_host, xdebug.remote_port, xdebug.remote_mode and xdebug.remote_handler" are all candidates to be controlled via the `$XDEBUG_CONFIG` environment variable, would it make sense to move those out of the config file and into an ENV declaration on the Dockerfile? This would further serve to demonstrate how Docker images and compose files can play with it.